java/src/jmri/Manager.java

```diff
@@ -608,7 +608,8 @@ default List<NamedBeanPropertyDescriptor<?>> getKnownBeanProperties() {
     static final int LIGHTS = TURNOUTS + 10;
     static final int REPORTERS = LIGHTS + 10;
     static final int MEMORIES = REPORTERS + 10;
-    static final int SENSORGROUPS = MEMORIES + 10;
+    static final int CLOCKS = MEMORIES + 10;
```

Comment:
I think Lights can use Clocks, e.g. for their scheduling, so it might be wise to put Clocks before Lights in the storage sequence.  Or do clocks depend on something else?

Reply:
SimpleTimebase depends on memories and sensors since it creates the memory IMCURRENTTIME and the sensor ISCLOCKRUNNING.

But I can move clocks to the beginning if that's better.

Reply:
I have updated the PR now to have clocks first, before sensors.

Reply:
> SimpleTimebase depends on memories and sensors since it creates the memory IMCURRENTTIME and the sensor ISCLOCKRUNNING.

I think that's OK.  The internal sensor manager is present from the beginning, so those can be created when needed.